app/invocation/invocation_share_button.tsx

```diff
@@ -106,11 +130,19 @@ export default class InvocationShareButtonComponent extends React.Component<
 
     return (
       <>
-        <FilledButton className="invocation-share-button" onClick={this.onShareButtonClick.bind(this)}>
-          {/* TODO: Use an icon that signifies the current permissions */}
-          <Share2 className="icon white" />
-          Share
-        </FilledButton>
+        {this.state.showingCopied && (
+          <OutlinedButton className="invocation-share-button">
+            <ClipboardCheck className="icon grey" />
+            <div class="invocation-share-button-text">Copied</div>
+          </OutlinedButton>
+        )}
```

Comment:
We have a few different places in the UI now that have custom "Copied" states like this.

I think we could simplify the code and unify the UX by calling `alert_service.success("Copied invocation link to clipboard")` which will show a banner for a second or two. We could even put the alert in `app/util/clipboard.ts > copyToClipboard()` and give it a second argument like `notification` which defaults to a generic "Copied to clipboard!" message. Could also add a special alert type like `clipboard` to give it a nicer icon / background, if you feel like it

Reply:
Ahh, didn't see that. Changed to use it and backed out the outlined button changes.